From 5022ad273a751e32c5ad7ba48cb44745cff24b52 Mon Sep 17 00:00:00 2001
From: Liu Guoli <liuguoli@mprc.pku.edu.cn>
Date: Wed, 26 Oct 2011 19:27:04 +0800
Subject: [PATCH 038/641] UniCore64: Select config CONFIG_GENERIC_HWEIGHT

Fixed error:
cfq-iosched.c:(.text+0xf98d): undefined reference to `__sw_hweight32'
lib/built-in.o: In function `__bitmap_weight': (.text+0x1c9c): undefined reference to `__sw_hweight64'

Signed-off-by: Liu Guoli <liuguoli@mprc.pku.edu.cn>
---
 arch/unicore64/Kconfig |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/unicore64/Kconfig b/arch/unicore64/Kconfig
index 029a0d0..1cff1ef 100644
--- a/arch/unicore64/Kconfig
+++ b/arch/unicore64/Kconfig
@@ -20,6 +20,9 @@ config ARCH_DMA_ADDR_T_64BIT
 config RWSEM_GENERIC_SPINLOCK
 	def_bool y
 
+config GENERIC_HWEIGHT
+	def_bool y
+
 config GENERIC_CALIBRATE_DELAY
 	def_bool y
 
-- 
1.7.9.5

